Index: b/autotests/client/CMakeLists.txt
===================================================================
--- a/autotests/client/CMakeLists.txt
+++ b/autotests/client/CMakeLists.txt
@@ -29,8 +29,6 @@
         )
     add_executable(testWaylandFullscreenShell ${testWaylandFullscreenShell_SRCS})
     target_link_libraries( testWaylandFullscreenShell Qt5::Test KF5::WaylandClient Wayland::Client)
-    add_test(NAME kwayland-testWaylandFullscreenShell COMMAND testWaylandFullscreenShell)
-    ecm_mark_as_test(testWaylandFullscreenShell)
 endif()
 
 ########################################################
@@ -75,8 +73,6 @@
         )
     add_executable(testWaylandSeat ${testWaylandSeat_SRCS})
     target_link_libraries( testWaylandSeat Qt5::Test Qt5::Gui KF5::WaylandClient KF5::WaylandServer Wayland::Client Wayland::Server)
-    add_test(NAME kwayland-testWaylandSeat COMMAND testWaylandSeat)
-    ecm_mark_as_test(testWaylandSeat)
 endif()
 
 ########################################################
